src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -27,26 +27,22 @@ export default class SentryApplicationRow extends React.PureComponent {
     onUninstall: PropTypes.func,
     onRemoveApp: PropTypes.func,
     onPublishRequest: PropTypes.func,
-    showPublishStatus: PropTypes.bool,
-    isInternal: PropTypes.bool,
-    hideButtons: PropTypes.bool,
+    showInstallationStatus: PropTypes.bool, //false if we are on the developer settings page where we don't show installation status
```

Comment:
I cleaned up the props on this component since we really one need one depending on if this is on the developer settings page or the integrations page. The rest of the logic is determined from the parameters of the app itself.